openrtb/macrosubs_test.go

```diff
@@ -6,106 +6,98 @@ import (
 	"github.com/Vungle/vungo/openrtb"
 )
 
-// verifyMacroSubs runs openrtb.MacroSubs and checks if the output is the expected string.
-func verifyMacroSubs(t *testing.T, input string, bidRes *openrtb.BidResponse, expectedOutput string) {
-	result, err := openrtb.MacroSubs(input, bidRes)
-	if err != nil {
-		t.Fatal("MacroSubs should complete successfully")
-	}
+type auctionResult struct {
+	price float64
+}
 
-	if result != expectedOutput {
-		t.Errorf("Got the wrong result.\nExpected: %s\nActual: %s", expectedOutput, result)
-	}
+func (ar *auctionResult) Price() float64 {
+	return ar.price
 }
 
-// TestMacroSubs creates a number of tests cases and passes them to verifyMacroSubs.
 func TestMacroSubs(t *testing.T) {
```

Comment:
Can you add a test in this function or a new one where it confirms that settlement.price is being used instead of the bid.price?